### PR TITLE
frontend: don't 500 for KRB5_USER_DENYLIST_PATTERNS=None

### PR DESCRIPTION
**Pull request by praiskup**
The default KRB5_USER_DENYLIST_PATTERNS=None is not iterable.

### CODE REVIEW

File: frontend/coprs_frontend/coprs/views/apiv3_ns/apiv3_general.py [hunk 1]
```diff
@@ -47,7 +47,7 @@ def krb_straighten_username(krb_remote_user):
     if not re.match(r"^[\w.-]+$", username):
         return None
 
-    for pattern in app.config.get("KRB5_USER_DENYLIST_PATTERNS", []):
+    for pattern in app.config.get("KRB5_USER_DENYLIST_PATTERNS") or []:
```
Comment (FrostyX) [inline frontend/coprs_frontend/coprs/views/apiv3_ns/apiv3_general.py:50]: You could probably do `app.config.get("KRB5_USER_DENYLIST_PATTERNS", [])` but it's not even worth changing.

**Review by FrostyX — APPROVED** [review]

File: frontend/coprs_frontend/coprs/views/apiv3_ns/apiv3_general.py (hunk 1 above)
Comment (praiskup) [reply frontend/coprs_frontend/coprs/views/apiv3_ns/apiv3_general.py:50]: Nah, that's what we had before..?

Comment (FrostyX) [reply frontend/coprs_frontend/coprs/views/apiv3_ns/apiv3_general.py:50]: /me dumb 
I understand the problem now.

Comment (praiskup) [reply frontend/coprs_frontend/coprs/views/apiv3_ns/apiv3_general.py:50]: np, thank you for the review!

**Pull request merged**